validator/controller.go

```diff
@@ -53,6 +55,8 @@ type controller struct {
 
 	validatorsMap       map[string]*Validator
 	newValidatorSubject pubsub.Subject
+
+	operatorPrivateKeyProvider eth1.OperatorPrivateKeyProvider
```

Comment:
@amir-blox could be a bit confusing with the operator admin key.
Maybe call it ShareEncryptionKey..?